### ISSUE TITLE
Feature Request: http.auth.basic for Docker services

### ISSUE THREAD

**Issue by derphilipp** [containous/traefik#971]
It would be awesome if an equivalent of
```toml
[entryPoints]
  [entryPoints.http]
  address = ":80"
  [entryPoints.http.auth.basic]
  users = ["test:$apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/"]
```
would be available to use with Docker services using labels, so that even "unsafe" services can be at least protected with a simple username/password gate.

Thanks for all of your hard work - I really do enjoy traefik quite a lot!

**Comment by Yshayy** [comment]
Hi, not sure if this issue is still relevant, but this feature is implemented in current docker provider (at least under swarm-mode) we use it for some of our services using the label:
traefik.frontend.auth.basic

It's documented here: https://docs.traefik.io/configuration/backends/docker/

**Comment by ldez** [comment]
Closed by #1147

**Comment by derphilipp** [comment]
Thanks!
